tc/python_worker/graphene/build_gsc_python_worker.sh

```diff
@@ -38,7 +38,7 @@ if [ "$GSC_IMAGE_EXISTS" = "yes" ]; then
 fi
 
 # Manifest files
-MANIFEST_FILE_DIR="${TCF_HOME}/examples/graphene_apps/python_worker/graphene/manifest"
+MANIFEST_FILE_DIR="${TCF_HOME}/tc/python_worker/graphene/manifest"
```

Comment:
$TCF_HOME/tc/graphene/python_worker/gsgx/manifest  (Note: gsgx is graphene-sgx)

Reply:
Ok